.coveragerc

```diff
@@ -0,0 +1,4 @@
+[report]
+show_missing = True
+omit =
+    *__init__*
```

Comment:
Just add new lines after this statement

Reply:
More color on this.

Reply:
The red arrow shows that you need to add a new line at the end of the file